controller/workitem_blackbox_test.go

```diff
@@ -2481,3 +2583,40 @@ func (s *workItemChildSuite) TestWorkItemListFilterByNoParents() {
 	})
 
 }
+
+func createSpaceWithDefaults(ctx context.Context, db *gorm.DB) (*space.Space, *iteration.Iteration, *area.Area) {
+	spaceRepo := space.NewRepository(db)
+	iterationRepo := iteration.NewIterationRepository(db)
+	areaRepo := area.NewAreaRepository(db)
+
+	newSpace := space.Space{
+		Name: fmt.Sprintf("The Space %v", uuid.NewV4()),
+	}
+	sp, err := spaceRepo.Create(ctx, &newSpace)
+	if err != nil {
+		fmt.Println("Failed to create space for area.")
```

Comment:
you have to use our logging package log.Error(....) as explained below. In this case, you can add the "space_id": as a property to the log instead of area_name